std/numeric.d

```diff
@@ -1925,6 +1925,7 @@ ElementType!Range entropy(Range)(Range r) if (isInputRange!Range)
     Unqual!(typeof(return)) result = 0.0;
     foreach (e; r)
     {
+        writeln(e);
```

Comment:
As said I experienced weird errors with this. I haven't got the time to look deeper in them, but the error from Travis is reproducible with `make - f posix.mak std.numeric.test`.